WordPress/gutenberg
# REST API: Use batch endpoint included in Core

Updates the default batch processor used by the `__experimentalBatch` action to use the batch endpoint that is included in WordPress 5.6 (`/batch/v1`) and not the one that Gutenberg includes (`/v1/batch`).

To test:

1. Go to Appearance → Widgets.
2. Modify a few blocks.
3. Press _Update_.
4. Verify that the changes were saved.
